### PR TITLE
UHF Commands Update

### PR DESCRIPTION
**Pull request by ArashYzd**
New UHF commands for the added subservices were added. CommandParser updated to allow the new arguments.

### CODE REVIEW

File: src/commandParser.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def __argCheck(self, args, inoutInfo, subservice=None):
 
     def __lexer(self, input):
         tokenList = []
-        tmp = re.split(r'([a-zA-Z_-]+|[\(\)]|[0-9_.-]*)', input)
+        tmp = re.split('([-.]+|[0-9.]+|[a-zA-Z0-9_-]+|[\(\)])', input)
```
Comment (ArashYzd) [inline src/commandParser.py:155]: 1. `autopep8` inserts `r` at the beginning which messed up the whole thing. So, I removed it.
2. The first bracket is for morse message. The second for float numbers. The third for alphanumerical arguments. The order must be as this so the command parser works correctly. However, please check and let me know if it ruins some other part of the code.

File: src/system.py [hunk 2]
```diff
@@ -315,18 +315,249 @@ def __init__(self):
...
+                            }
+                        }
+                    },
+                    'UHF_GET_BEACON_MSG': {
+                        'subPort': 40,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'Beacon Message': '>U36',  # 120 too large
```
Comment (ArashYzd) [inline src/system.py:545]: For all of the beacon and call sign stuff, the EH allocates a length of 120. However, based on the manual, other than this one, the rest will be of length 6 or 36. However, when you allocate 120 for this one, it'll go beyond the buffer size and things do not work out. The best solution would be to somehow read the length of the message (similar to sending it in `UHF_SET_BEACON_MSG` automatically and set the length using that which I don't know how to do.

Also, the command parser doesn't allow spaces in the middle of the message which is not good. Any suggestions for that?

File: src/system.py [hunk 3]
```diff
@@ -315,18 +315,249 @@ def __init__(self):
...
+                                'FRAM': '>u1',
+                                'SEC': '>u1',
+                                'CTS': '>u1',
+                                'Boot': '>u1',
+                                'Pipe': '>u1',
+                                'BCN': '>u1',
+                                'Echo': '>u1',
+                                'RF Mode1': '>u1',  # Use table 9 to improve
+                                'RF Mode2': '>u1',
+                                'RF Mode3': '>u1',
```
Comment (ArashYzd) [inline src/system.py:490]: The RF Mode is like `110` or `010`. However, the status control is an array[12]. I couldn't figure out how to send 3 consecutive `uint8_t` as one. So, I improvised. Also, I should check with Thomas why there are 12. The manual shows more.

File: src/system.py (hunk 2 above)
Comment (arrooney) [reply src/system.py]: In the middle of the message, as in a string? Should be able to add a space into the regular expression

File: src/commandParser.py (hunk 1 above)
Comment (arrooney) [reply src/commandParser.py:155]: As long as you've tested it and the commands still work as expected, it should be fine

File: src/system.py (hunk 3 above)
Comment (arrooney) [reply src/system.py:513]: Yeah, I hadn't even thought of arrays in the return type, This should be good enough for now at least.

Comment (arrooney) [reply src/system.py:513]: Actually, I'm a bit confused about this. Perhaps you can explain at the work session

**Comment by ArashYzd** [comment]
@arrooney I'm planning to work on the test scripts and I noticed this PR wasn't merged. I'm pretty sure we talked about the issues over the meetings especially since the last commits were for documentation. So, there shouldn't be a problem other than the new conflicts.
Please let me know what's the best way to merge these commits atm.

**Review by arrooney — COMMENTED** [review]
LGTM

**Pull request merged**